src/main/java/picard/vcf/LiftoverVcf.java

```diff
@@ -313,21 +323,155 @@ public static void main(final String[] args) {
         return 0;
     }
 
-    protected static GenotypesContext fixGenotypes(final GenotypesContext originals, final Map<Allele, Allele> reverseComplementAlleleMap) {
+    protected static Locatable getFlippedLocation(final VariantContext source, final LiftOver liftOver) {
+        Interval originalLocus = new Interval(source.getContig(), source.getStart() + 1, source.getEnd() + 1);
+        return liftOver.liftOver(originalLocus);
+    }
+
+    /**
+     * @param source            original variant context
```

Comment:
would be good to explain that it's used in a context where the old and new seqs are on different strands